src/main/java/com/snowflake/kafka/connector/internal/SnowflakeSinkServiceV1.java

```diff
@@ -311,6 +329,17 @@ public void setBehaviorOnNullValuesConfig(
     this.behaviorOnNullValues = behavior;
   }
 
+  // enable use of new stage files processor
+  void enableStageFilesProcessor() {
+    this.useStageFilesProcessor = true;
+    if (null != cleanerServiceExecutor) {
```

Comment:
Please use a consistent way of null checking - I think in other places we use variable != null